RELEASING.md

```diff
@@ -106,16 +113,88 @@ done
 
 Check your e-mail and make sure the number of “you’ve published this module” emails matches the number you expect.
 
-## Publish the GitHub Release
+### Publish the GitHub Release
 
 Publish the GitHub release, ensuring that the tag points to the newly landed commit corresponding to release proposal `x.y.z`.
 
-## Update CHANGELOG
+### Update CHANGELOG
 
 - After releasing is done, update the [CHANGELOG.md](https://github.com/open-telemetry/opentelemetry-js/blob/master/CHANGELOG.md) and start new Unreleased label.
 - Create a new commit with the exact title: `Post Release: update CHANGELOG.md`.
 - Go through PR review and merge it to GitHub master branch.
 
-## Known Issues
+### Known Issues
 
 - The `examples/` and `getting-started/` folders are not part of lerna packages, we need to manually bump the version in `package.json`.
+
+## Versioning
+
+### Assumptions
+
+This project uses semver v2, as does the rest of OpenTelemetry.
+
+### Goals
+
+#### API Stability
+
+Once the API for a given signal (spans, logs, metrics, baggage) has been officially released, that API module will function with any SDK that has the same major version, and equal or greater minor or patch version.
+
+For example, libraries that are instrumented with `@opentelemetry/api@1.0.1` will function with SDK library `@opentelemetry/node@1.11.33` or `@opentelemetry/web@1.3.4`
+
+#### SDK Stability:
+
+Public portions of the SDK (constructors, configuration, end-user interfaces) must remain backwards compatible.
+Internal interfaces are allowed to break.
+
+### Methods
+
+#### Mature Signals
+
+API modules for mature (i.e. released) signals will be included in the `@opentelemetry/api` module.
+
+#### Immature or experimental signals
+
+API modules for experimental signals will not be included in the `@opentelemetry/api` module, and must be installed manually. API modules will be named with an `-experimental` suffix to make it abundantly clear that depending on them is at your own risk. For example, the `@opentelemetry/metrics-api-experimental` module will provide experimental access to the unfinished metrics API. NO STABILITY GUARANTEES ARE MADE.
```

Comment:
I thought we agreed to remove the `-experimental` suffix at the SIG meeting. It looks like the "versioning" section was updated, but this might have been missed.

Reply:
Fixed